### PR TITLE
Fixes angular tutorial

### PR DESCRIPTION
**Pull request by igaldino**
A few fixes to Angular tutorial steps 3 and 5.

### CODE REVIEW

**Review by janchild — APPROVED** [review]
LGTM!

File: src/pages/developing/angular-tutorial/step-5.mdx [hunk 1]
```diff
@@ -152,7 +152,7 @@ first.
 npm run build
 ```
 
-Looking at `package.json`, you'll find `yarn build` to run `ng build`. This
+Looking at `package.json`, you'll find `ng build`. This
```
Comment (cal-smith) [inline src/pages/developing/angular-tutorial/step-5.mdx:155]: Just curious why the switch away from `yarn build`? `npm run build` also seems a better choice

File: src/pages/developing/angular-tutorial/step-5.mdx [hunk 2]
```diff
@@ -262,10 +262,10 @@ sure we only deploy immediately after running the build. In the `"scripts"`
 object in `package.json`, add:
 
 ```bash path=package.json
-"deploy": "rm -rf ./dist && yarn build && cf push -f manifest.yml"
+"deploy": "rm -rf ./dist && ng build --prod --aot && cf push -f manifest.yml"
 ```
 
-Next time you want to deploy, you can simply run `yarn deploy`.
+Next time you want to deploy, you can simply run `npm run-script deploy`.
```
Comment (cal-smith) [inline src/pages/developing/angular-tutorial/step-5.mdx:268]: Same question as above, I believe we use `yarn` everywhere else, so the change is strange. If we want to go with `npm` instead this should be `npm run deploy`

**Review by cal-smith — CHANGES_REQUESTED** [review]
Thanks for taking the time to update this! Just a couple questions, but it seems good otherwise!

File: src/pages/developing/angular-tutorial/step-5.mdx (hunk 1 above)
Comment (igaldino) [reply src/pages/developing/angular-tutorial/step-5.mdx:155]: Sure, in the tutorial, this is the only mention to yarn.

File: src/pages/developing/angular-tutorial/step-5.mdx (hunk 2 above)
Comment (igaldino) [reply src/pages/developing/angular-tutorial/step-5.mdx:268]: In the tutorial, this is the second mention to yarn.

About `npm run deploy`, I didn't try but `npm run-script deploy` works just fine too.

Comment (igaldino) [reply src/pages/developing/angular-tutorial/step-5.mdx:268]: the rest of angular tutorial uses ng or npm

Comment (cal-smith) [reply src/pages/developing/angular-tutorial/step-5.mdx:268]: Ah, my mistake on the `yarn` situation ... This seems fine then

**Review by tw15egan — APPROVED** [review]

**Pull request merged**